vcbuild.bat

```diff
@@ -86,8 +92,8 @@ if /i "%1"=="noetw"         set noetw=1&goto arg-ok
 if /i "%1"=="ltcg"          set ltcg=1&goto arg-ok
 if /i "%1"=="licensertf"    set licensertf=1&goto arg-ok
 if /i "%1"=="test"          set test_args=%test_args% -J %common_test_suites%&set lint_cpp=1&set lint_js=1&set lint_md=1&goto arg-ok
-:: test-ci is deprecated
-if /i "%1"=="test-ci"       goto arg-ok
```

Comment:
The removal of the test-ci target breaks some of the CI jobs, see https://ci.nodejs.org/job/node-test-commit-windows/976/nodes=win10/console:
```
C:\workspace\node-test-commit-windows\nodes\win10>vcbuild.bat release nosign x64 test-ci ignore-flaky NODE_TEST_DIR=C:\node-tmp  
Note: vcbuild no longer signs by default. "nosign" is redundant.
Error: invalid command line option `test-ci`.
```
(Apologies if it's already been reported. I searched nodejs/node and nodejs/build but didn't find an exact match.)

Reply:
@bnoordhuis any reason not to use `node-test-commit-windows-fanned` instead? `node-test-commit-windows` has been forgotten for a long time, I'll archive it if there's no objection.

Note that `test-ci` was a no-op as removed here, running with it only compiled, did not run tests.

Reply:
Ah, if it's unmaintained then yes, please remove or archive it to avoid confusion. :-)